### PR TITLE
Remove Signals and QueryableState

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: docs/MigrationGuide.md [hunk 1]
```diff
@@ -3,7 +3,15 @@
 
 To see the biggest differences please consult the [changelog](Changelog.md).
 
-## In version 1.7.0 (Not released yet)
+## In version 1.8.0
+           
+* [#3835](https://github.com/TouK/nussknacker/pull/3835) Remove Signals and QueryableState. This change affects:
```
Comment (lciolecki) [inline docs/MigrationGuide.md:8]: Removed? :)

File: designer/server/src/test/scala/pl/touk/nussknacker/ui/integration/BaseFlowTest.scala [hunk 2]
```diff
@@ -139,6 +139,7 @@ class BaseFlowTest extends AnyFunSuite with ScalatestRouteTest with FailFastCirc
 
       val (relevant, other) = settings.partition { case (k, _) => underTest.keySet contains k }
       relevant shouldBe underTest
+      println(other.values.filter(_.docsUrl.isDefined))
```
Comment (gskrobisz) [inline designer/server/src/test/scala/pl/touk/nussknacker/ui/integration/BaseFlowTest.scala:142]: Is this `println` necessary?

**Review by gskrobisz — APPROVED** [review]

File: designer/server/src/test/scala/pl/touk/nussknacker/ui/integration/BaseFlowTest.scala (hunk 2 above)
Comment (mproch) [reply designer/server/src/test/scala/pl/touk/nussknacker/ui/integration/BaseFlowTest.scala]: :blush: fixed...

File: docs/MigrationGuide.md (hunk 1 above)
Comment (mproch) [reply docs/MigrationGuide.md]: :+1:

**Pull request merged**